test/test-imports/typescript/test.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/sh
```

Comment:
why?

Reply:
Based on some of your previous comments, I thought I'd remove it. It's a `.sh` file and the shebang seems redundant here. Open to putting it back though.

Reply:
After putting it back, seems to cause this error on ubuntu:

```
python/test.sh: 6: python/test.sh: source: not found
```

Going to revert back to having it removed.